### DISCUSSION TITLE
I wish that clipious have pc version.

### DISCUSSION THREAD

**Discussion by DrVektor** [lamarios/clipious#329] [Ideas]
Clipiuse spetsifikatsioonid on nii head. Kahjuks ei ole PC-l samaväärset. Ma soovin, et oleks olemas clipious'i versioon, millel on kõik funktsioonid ja seaded, kuid mis töötab windowsis. Aga ei ole. ☹️. Aga ma usun. saab olema. Sest väga hea programm ei saa sobida ainult android maailmas.

**Comment by lamarios** [comment]
It should be actually doable without too much effort. The only blocker I can think of is the video player, but using something like media_kit as a video library should make it work. 

Though while i could build a linux version, i don't have a windows machine to do so.

Not gonna lie, I don't have much time to maintain another platform as well as multiple new build systems (windows, linux etc...). Feel free to contribute.

**Reply by DrVektor** [reply]
Hmm. Thank you very much for getting back to me. Also, thank you, you have developed a great programme. So, the video player api like in the android version is not possible in window. The video player in the Freetube example is very cumbersome and useless. For example, it is useless on a slow internet connection.  I hope someone will contribute something. Again, health to your labour. Thank you very much.

**Comment by Vikash-Singh-Tanwar** [comment]
> It should be actually doable without too much effort. The only blocker I can think of is the video player, but using something like media_kit as a video library should make it work.
> 
> Though while i could build a linux version, i don't have a windows machine to do so.
> 
> Not gonna lie, I don't have much time to maintain another platform as well as multiple new build systems (windows, linux etc...). Feel free to contribute.

i am a student learning coding know a bit as well if you can give instruction how to build it ill maintain for windows and if you know anything about tizen tv os we con make it for that as well ill do the handy work just instruct me
